cmd/agent/openstack_iid_attestor/main.go

```diff
@@ -27,16 +26,12 @@ import (
 // IIDAttestorPlugin implements the nodeattestor Plugin interface
 type IIDAttestorPlugin struct {
 	logger   hclog.Logger
-	config   *IIDAttestorPluginConfig
 	metaData *openstack.Metadata
 
-	mtx *sync.RWMutex
-
 	getMetadataHandler func() (*openstack.Metadata, error)
 }
 
 type IIDAttestorPluginConfig struct {
-	trustDomain string
 }
```

Comment:
Since the parameter is not defined in the agent plugin config,
I wonder if we should remove this struct, What do you think?

Reply:
fix: 0fe4d7e